Rubberduck.Parsing/Symbols/Attributes.cs

```diff
@@ -236,7 +229,7 @@ public bool HasExposedAttribute(out AttributeNode attribute)
         /// <summary>
         /// Corresponds to *not* having the TYPEFLAG_FNONEXTENSIBLE flag set (which is the default for VBA).
         /// </summary>
-        public void AddExtensibledClassAttribute()
+        public void AddExtensibleClassAttribute()
         {
             Add(new AttributeNode("VB_Customizable", new[] { "True" }));
```

Comment:
Nitpick, but IIR we've been using the lexer tokens elsewhere - `Tokens.True`?

Reply:
The literal already got used in the old versions of the attribute annotations. However, I will change that as your suggestion is cleaner. (The compiler probably inclines it anyway.)